src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -331,13 +332,14 @@ def _create_osd(self, svc_arg=None, inbuf=None):
 
         usage = """
 Usage:
-  ceph orchestrator osd create -i <json_file>
+  ceph orchestrator osd create -i <json_file/yaml_file>
   ceph orchestrator osd create host:device1,device2,...
 """
 
         if inbuf:
             try:
-                drive_group = DriveGroupSpec.from_json(json.loads(inbuf))
+                dgs = DriveGroupSpecs(yaml.load(inbuf))
```

Comment:
`yaml.safe_load(inbuf)` is suggested (https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation).

Just wondering if it's possible to hit some corner case like this:  https://stackoverflow.com/a/51446265  (But I *can't* reproduce this error)

Reply:
I think for the moment it's good enough.